vinaigrette.py

```diff
@@ -0,0 +1,29 @@
+import datetime
+import random
+
+
+def random_date_in_range():
+    date1 = input("Enter first date (YYYY-MM-DD):")
```

Comment:
first? second? you are subtracting one from the other to get a delta, and then make sure the first date is before the second date . If so, why not give them names that indicate what they are? start_date and second_date are way better